### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by mremolt**
The injection token is just called 'Props' for every type now.

# New Pull Request

For more information, see the `CONTRIBUTING` guide.

## Prerequisites

Please make sure you can check the following boxes:

- [ ] I have read the **CONTRIBUTING** document
- [ ] My code follows the code style of this project
- [ ] All new and existing tests passed

### Contribution Type

What types of changes does your code introduce? Put an `x` in all the boxes that apply:

- [ ] Bug fix (non-breaking change which fixes an issue, please reference the issue id)
- [ ] New feature (non-breaking change which adds functionality, make sure to open an associated issue first)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] I have updated the documentation accordingly
- [ ] I have added tests to cover my changes

### Description

[Place a meaningful description here]

### Remarks

[Optionally place any follow-up comments, remarks, observations, or notes here for future reference]

### CODE REVIEW

File: src/converters/piral-ng/README.md [hunk 1]
```diff
@@ -166,17 +166,17 @@ Depending on your Angular needs you'd want to share more packages.
...
-| Component | Props            | Piral   | Context   |
-|-----------|------------------|---------|-----------|
-| Tile      | `TileProps`      | `Piral` | `Context` |
-| Page      | `PageProps`      | `Piral` | `Context` |
-| Modal     | `ModalProps`     | `Piral` | `Context` |
-| Extension | `ExtensionProps` | `Piral` | `Context` |
-| Menu      | `MenuProps`      | `Piral` | `Context` |
+| Component | Props   | Piral   | Context   |
+|-----------|---------|---------|-----------|
+| Tile      | `Props` | `Piral` | `Context` |
```
Comment (FlorianRappl) [inline src/converters/piral-ng/README.md:171]: I think the `Piral` is actually `piral`. Can you confirm (and if I'm right change it)? Thanks!

File: src/converters/piral-ng/README.md [hunk 2]
```diff
@@ -166,17 +166,17 @@ Depending on your Angular needs you'd want to share more packages.
...
+| Page      | `Props` | `Piral` | `Context` |
+| Modal     | `Props` | `Piral` | `Context` |
+| Extension | `Props` | `Piral` | `Context` |
+| Menu      | `Props` | `Piral` | `Context` |
 
 To use such a service the `@Inject` decorator should be used with the explicit name.
 
-The following code snippet illustrates the injection of the `TileProps` service into a sample tile component.
+The following code snippet illustrates the injection of the `Tile Props` service into a sample tile component.
```
Comment (FlorianRappl) [inline src/converters/piral-ng/README.md:179]: Maybe have it rephrased as "...injection of the `Props` service from an `TileProps` interface into a sample tile component."

**Review by FlorianRappl — COMMENTED** [review]
Thanks! :beers:

File: src/converters/piral-ng/README.md (hunk 1 above)
Comment (mremolt) [reply src/converters/piral-ng/README.md]: You are right, changed it.

File: src/converters/piral-ng/README.md (hunk 2 above)
Comment (mremolt) [reply src/converters/piral-ng/README.md]: Done

**Pull request merged**